web/Dockerfile.travis

```diff
@@ -29,6 +29,7 @@ RUN wget --no-check-certificate https://bootstrap.pypa.io/ez_setup.py -O - | pyt
 RUN easy_install-2.7 supervisor
 RUN useradd -s /bin/bash -u 3000 -m impact_user
 RUN pip3 install -r /wwwroot/requirements/travis.txt
+RUN pip3 install --no-dependencies -r /wwwroot/requirements/no_dependencies.txt
```

Comment:
I saw the `no_dependencies.txt` file before I saw this line, and had thought maybe the file was for packages which did not have any dependencies.  Since a requirements file paired with a special pip invocation is an unusual arrangement for requirements in our Python code, I'd like to see a comment at the top of the `no_dependencies.txt` file clarifying that the packages listed will be installed with pip's `--no-dependencies` flag.

(I'm also curious why we need to install `django-graphql-jwt` in this way, but that's just curiosity)

Reply:
@pbx the reason is that django-graphql-jwt is supported only for django 1.11+. I did not run into any actual issue with using it in django 1.10, except for the fact that it tries to install a newer django version than we'd like. 

I'll add the comment you requested.